modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RankEvalRequestIT.java

```diff
@@ -98,7 +107,9 @@ public void testPrecisionAtRequest() {
                 RankEvalAction.INSTANCE, new RankEvalRequest());
         builder.setRankEvalSpec(task);
 
-        RankEvalResponse response = client().execute(RankEvalAction.INSTANCE, builder.request().indices("test"))
+        // String indexToUse = randomBoolean() ? TEST_INDEX : INDEX_ALIAS;
```

Comment:
@cbuescher  With this commented out string, did you intend to use either `TEST_INDEX` or `INDEX_ALIAS`?

Reply:
Yes, I think I was intending to randomize this and commented it out for some reason, will correct this.